content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -725,7 +725,7 @@ See the [resourceQuota design doc](https://git.k8s.io/community/contributors/des
 
 ### RuntimeClass {#runtimeclass}
 
-{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
```

Comment:
Shouldn't this be v1.20 stable? The admission controller is used for runtimeclass scheduling, not just pod overhead.

Reply:
Yeah... I was thinking about it. Making it stable creates false impression that PodOverhead is also stable. @sftim what would you suggest? RuntimeClass feature is stable, PodOverhead which is part of the same API is in beta

Reply:
This is exactly what I asked on the last SIG node meeting as I also lack context on how it is typically handled in k8s. So the outcome of the discussion was that this is fine. It is similar to the situation when the new feature being enabled via feature gate on the already stable API. So the text you suggested looks accurate

Reply:
```suggestion
{{< feature-state for_k8s_version="v1.20" state="stable" >}}
```